__tests__/app.test.js

```diff
@@ -181,24 +218,23 @@ describe("/api/articles/:article_id/comments", () => {
             author: "butter_bridge",
             body: "what a wonderful day",
             article_id: 1,
-            votes: 0, 
+            votes: 0,
             created_at: expect.any(String),
             comment_id: expect.any(Number),
-                });
           });
         });
-    
-  
-  test('400 responds with an appropriate status and error message when provided with a no comment', () => {
-    return request(app)
-      .post('/api/articles/1/comments')
-      .send({
-        username: "butter_bridge"
-      })
-      .expect(400)
-      .then(({body}) => {
-        expect(body).toEqual({message:"Bad request"});
-      });
+    });
+
+    test("400: responds with an appropriate status and error message when provided with a no comment", () => {
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send({
+          username: "butter_bridge",
+        })
+        .expect(400)
+        .then(({ body }) => {
+          expect(body).toEqual({ message: "Bad request" });
+        });
+    });
```

Comment:
Nice error testing. 

Another test you can add is if the username doesn't exist, you should only be able to post with a registered username!